sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java

```diff
@@ -1877,6 +1877,14 @@ public void onUpgrade(SQLiteDatabase db, ConnectionSource connectionSource, int
 				getDao(Event.class).executeRaw("ALTER TABLE events ADD COLUMN evolutionDate timestamp;");
 				getDao(Event.class).executeRaw("ALTER TABLE events ADD COLUMN evolutionComment text;");
 
+			case 265:
+				currentVersion = 265;
+				getDao(Facility.class).executeRaw("ALTER TABLE facility ADD COLUMN street varchar(4096);");
+				getDao(Facility.class).executeRaw("ALTER TABLE facility ADD COLUMN houseNumber varchar(255);");
```

Comment:
These new fields should use the same lengths as their equivalents in the Location entity because we might run into problems otherwise; as far as I can see, this affects houseNumber and postalCode (should be 512; also needs to be changed for the definition in sormas_schema.sql)